app/_src/gateway/reference/configuration.md

```diff
@@ -532,6 +591,49 @@ The SNI (Server Name Indication extension) to use for Vitals telemetry data.
 
 
 ---
+{% if_version gte:3.3.x %}
+### cluster_dp_labels
+{:.badge .enterprise}
+
+Comma separated list of Labels for the data plane.
+
+Labels are key-value pairs that provide additional context information for each
+DP.
+
+Each label must be configured as a string in the format `key:value`.
+
+Labels are only compatible with hybrid mode deployments with Kong Konnect
+(SaaS), this configuration doesn't work with self-hosted deployments.
+
+Keys and values follow the AIP standards: https://kong-aip.netlify.app/aip/129/
+
+Example: `deployment:mycloud,region:us-east-1`
+
+Default: none
+
+---
+
+### cluster_dp_labels
+{:.badge .enterprise}
+
+Comma separated list of Labels for the data plane.
+
+Labels are key-value pairs that provide additional context information for each
+DP.
+
+Each label must be configured as a string in the format `key:value`.
+
+Labels are only compatible with hybrid mode deployments with Kong Konnect
+(SaaS), this configuration doesn't work with self-hosted deployments.
+
+Keys and values follow the AIP standards: https://kong-aip.netlify.app/aip/129/
+
+Example: `deployment:mycloud,region:us-east-1`
+
+Default: none
+
+---
```

Comment:
This entry appears twice. Looks like the issue is the source file though, that'll need fixing.

```suggestion
```